docs/changelog.txt

```diff
@@ -46,6 +46,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## API
 
 ## Lua
+- `tile-material`: fix the order of declarations. The GetTileMat function now returns the material as intended (always returned nil before). Also changed the license info, with permission of the original author.
```

Comment:
Needs to be double-backticks because there isn't an anchor of this name

From https://github.com/DFHack/dfhack/runs/6971381872?check_suite_focus=true

```
docs/_auto/news.rst:6: WARNING: undefined label: tile-material
writing output... [ 33%] docs/Authors .. docs/NEWS
docs/_auto/news-dev.rst:6: WARNING: undefined label: tile-material
```

Fix:
```suggestion
- ``tile-material``: fix the order of declarations. The GetTileMat function now returns the material as intended (always returned nil before). Also changed the license info, with permission of the original author.
```

Reply:
Also could use double backtics around the function name